code/go/0chain.net/smartcontract/dbs/event/blobber.go

```diff
@@ -121,6 +121,16 @@ func (edb *EventDb) addOrOverwriteBlobber(blobber Blobber) error {
 	return result.Error
 }
 
+func (bl *Blobber) existsV2(edb *EventDb) (bool, error) {
```

Comment:
Lets not create a new function & revamp the older one

Reply:
Done

Reply:
yaa my bad forgot to do that was actually planning to that only 😓
Np fixed that